latent_space/model_latent.py

```diff
@@ -0,0 +1,380 @@
+__author__ = "Keenan Manpearl"
```

Comment:
I'm a bit confused why you need this script? Doesn't it exist somewhere else in this repo? Would it be possible to import from there (instead of, presumably, duplicating the code)?